bom/pom.xml

```diff
@@ -620,20 +591,6 @@
                 <version>${neo4jScalaLibraryVersion}</version>
             </dependency>
 
-            <!-- Cassandra -->
-            <dependency>
-                <groupId>com.datastax.cassandra</groupId>
-                <artifactId>cassandra-driver-core</artifactId>
-                <version>${cassandraVersion}</version>
-            </dependency>
-
-            <!-- Redis -->
-            <dependency>
-                <groupId>biz.paluch.redis</groupId>
-                <artifactId>lettuce</artifactId>
-                <version>${lettuceVersion}</version>
-            </dependency>
-
             <!-- RESTEasy for accessing CouchDB -->
             <dependency>
                 <groupId>org.jboss.resteasy</groupId>
```

Comment:
If we use it only for CouchDB, we should remove it. If not, we should change the comment.